mala/network/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .hyper_opt_naswot import HyperOptNASWOT
 from .hyper_opt_oat import HyperOptOAT
 from .predictor import Predictor
+from .multi_predictor import MultiPredictor
```

Comment:
Can this class be moved to the `predictor` module, or what is the reason for having a separate module?

Reply:
See above.

Reply:
Yes as above, it is one possible design decision (`module_method_a.ClassMethodA`).